docs/shared/ungrib.yaml

```diff
@@ -0,0 +1,13 @@
+ungrib:
+  execution:
+    mpicmd: srun
```

Comment:
I recall that when we tested with a `null` value here, the schema validator rightly complained. But it would be interesting to know if we can omit `mpicmd` entirely, and maybe reconsider our schema for the `execution` block (and potentially our Python code that accesses this value), since not all components we write drivers for will need -- or IMO should have to specify if they don't -- a command to run under MPI, because they're not MPI codes. `ungrib` is not an MPI code, for example.

Reply:
Quick follow-up on this one: Could you try running with this line removed and see what happens? It looks to me like it's not required by the schema, but the `Driver` code appears to require it. If that's the case, I propose that we leave this as-is for now, but I'll put a TODO on my list to make this more flexible, so that non-MPI codes don't need to specify an `mpicmd`. @christinaholtNOAA Any reason you can think of not to go this way?

Reply:
When I ran it without `mpicmd` I got a `KeyError` because it's needed in `_runcmd()`.

Reply:
I thought so. Thanks for testing. I think you can resolve this comment, and I'll take a TODO to make this flexible unless @christinaholtNOAA dissuades me.